{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributor Guide
+
+Thank you for your interest in improving this project.
+This project is open-source under the [MIT license] and
+welcomes contributions in the form of bug reports, feature requests, and pull requests.
+
+Here is a list of important resources for contributors:
+
+- [Source Code]
+- [Documentation]
+- [Issue Tracker]
+- [Code of Conduct]
+
+[mit license]: https://opensource.org/licenses/MIT
+[source code]: https://github.com/cjolowicz/hypermodern-python
+[documentation]: https://hypermodern-python.readthedocs.io/
+[issue tracker]: https://github.com/cjolowicz/hypermodern-python/issues
+
+# How to report a bug
```

Comment:
This heading needs to be second-level. Same for all headings after it.

```suggestion
## How to report a bug
```

This is important so the subheadings don't get displayed in the navigation sidebar in Sphinx.

Reply:
fixed: ab9b60411355c0504080d280084c96082862bf72

Tested result:
![image](https://user-images.githubusercontent.com/38903141/152373824-b884f5d9-0e25-44fd-a228-214602b551a7.png)